composer.json

```diff
@@ -42,7 +42,7 @@
         "symfony/polyfill-mbstring": "^1.28",
         "symfony/polyfill-php80": "^1.28",
         "symfony/polyfill-php81": "^1.28",
-        "symfony/process": "^5.4 || ^6.0 || ^7.0 <7.2",
+        "symfony/process": "^5.4 || ^6.0 || ^7.0",
```

Comment:
if someone would install 7.0, would test still pass?
if not, I suggest to bump requirement to ^7.2

(no big value for having different test case depends on dependency version)

Reply:
That is a small issue, as it would weaken `--prefer-lowest` tests in dependant packages.

Reply:
I prefer those packages to stay with v3.66 and upgrade when they are ready to upgrade symfony/*, or to switch to shim version - over developer of Fixer having tests pasting or not depending on deps resolution

Reply:
So, (put your own number here) developers using Unix systems will have to upgrade Symfony or not to upgrade PHP CS Fixer because of... Windows?

I imagine - as Symfony 7.0/7.1 are quite fresh - this could block quite a few people, and currently, there is no practical gain of blocking Symfony 7.0/7.1. If there will be a need to act, I'd go in the direction, where in the test we check if the command is `'php -l foo.php'` or `'c:\tools\php\php.EXE -l foo.php'`.

Reply:
if you want to do that extra check, go ahead.
but I don't believe we need to hyperfocus on those who can upgrade one library but cannot upgrade the 2nd. as especially for that (put your own number) fractal of users, we recommend to install shim